iina/FileGroup.swift

```diff
@@ -17,7 +17,6 @@ class FileInfo: Hashable {
   var filename: String
   var ext: String
   var nameInSeries: String?
-  var characters: [Character]
```

Comment:
That's my first thought but Instruments suggest otherwise. You know what, I'm beginning to doubt whether Instruments is pointing me to the right direction.